pages/common/cmark.md

```diff
@@ -0,0 +1,19 @@
+# cmark
+
+> Converts CommonMark Markdown formatted text to other formats.
+
+- Render a Commonmark Markdown file to HTML:
+
+`cmark --to html {{filename.md}}`
+
+- Render the standard input to latex:
```

Comment:
How about "Convert data from standard input to latex" ?

Reply:
Sure :-)